docs/logs/reports-and-components.md

```diff
@@ -0,0 +1,75 @@
+
+### App State
+
+When your Logs App is loaded the **Save and Reset buttons will be hidden** since you see the App's default state. If a **new component is added** or any existing **component is edited, alert created, and report component removed**, the **Save and Reset buttons will appear** in the report menu and allow for changes to be saved or reset to the initial state. If your browser is suddenly closed, and no changes were saved or reset, the App will preserve its initial state.
+
+### Report Actions 
+
+The set of icons in the logs search and report menu let you:
+
+- **Save Query / Add Alert** - Every query can be sa**ved for later use, and used to send alerts**. The ability to save queries and alerts is part of the essential tools for searching, correlating and investigating logs data. Each saved query becomes part of the saved queries section.
+
+- **Saved Queries / Alert Rules** - Right next to the save query / alert icon you can see the saved queries and alert rules, this will open a modal dialogue window with **all your saved queries and alert rules** displayed in a [Sematable](https://github.com/sematext/sematable), a table component that displays
+
+	- _Tag Color_ that can be assigned to any logs query while it is being saved by user using colour picker componenet 
+	- _Query Name_ to further identify and describe logs query
+	- _Query String_ will be string you use in logs search input field
+	- _Owner_	
+	- _10m, 1h, 12h, and 1d_ shortcuts with histogram for fast observability of query patterns across common time intervals
+	- Saved _query actions_ dropdown menu used to add new, edit, clone, reset, and delete saved logs report
+
+- **Email Report** - There are E-mail and Subscription types available with time, frequency, and time range defined so your team can get automatated reports about your logs and system health.
+
+- **Full Screen Mode** - Entering full screen mode brings only the component you choose into view. It's useful for presentations and viewing detailed info of a single component. All charts have zoom, hairline, and time navigation tools with tooltips available for effective chart interaction and data analysis.
+
+- **Connected Applications** - Connecting a Logs App to a Monitoring App will **pre-select that Logs App when you decide to correlate metrics from the connected Monitoring App**. When you receive an alert notification for an app, the **alert notification will automatically include information and charts from connected apps**, and thus provide more information and context for you. You can connect any two apps you have access to, regardless of their type. **Any App can be connected to any number of other Apps**.
+
+- **Integrations** - This section will guide you through installing Agents and Monitors. You will need to install **Sematext Agents so metrics or logs** data can be shipped to your Logs or Monitoring App. 
```

Comment:
Maybe we need to clarify more here.  To ship logs one doesn't have to install a Sematext agent.  One can use many other log libs or shippers.

Reply:
Changed this to be about log shippers instead of agents